http-common/src/server.rs

```diff
@@ -1,5 +1,6 @@
 // Copyright (c) Microsoft. All rights reserved.
 
+#[cfg(feature = "tokio1")]
```

Comment:
Just get rid of this feature entirely. We don't need it now that Gordon and Lee have completed the tokio v1 refactor of the iotedge repo.